packages/api/src/modules/user/user.service.test.ts

```diff
@@ -279,6 +286,72 @@ describe("User Service", () => {
         });
     });
 
+    describe("disable", () => {
+        const USER_ID = USER_WITHOUT_SECRET._id.toString();
+        // @ts-expect-error: private method
+        const mockAnonymize = jest.spyOn(userService, "anonymize");
+        beforeAll(() => {
+            // @ts-expect-error: private method
+            mockAnonymize.mockImplementation(() => ANONYMIZED_USER);
+        });
+
+        beforeEach(() => {
+            // @ts-expect-error: mock
+            userRepository.findById.mockReset();
+            // @ts-expect-error: mock
+            userRepository.update.mockReset();
+        });
+
+        it("should fetch user from db", async () => {
+            await userService.disable(USER_ID);
+            // @ts-expect-error: mock
+            userRepository.findById.mockImplementationOnce(jest.fn(async () => USER_WITHOUT_SECRET));
```

Comment:
```suggestion
            jest.mocked(userRepository.findById).mockImplementationOnce(jest.fn(async () => USER_WITHOUT_SECRET));
```

J'ai découvert ça, peut-être que ça peut épargner le `// @ts-expect-error`

Reply:
pas mal je ne connaissais pas